web_src/fomantic/css.js

```diff
@@ -0,0 +1,260 @@
+    css(callback);
+  }
+}
+
+function docs(src, callback) {
+  if (callback === undefined) {
+    callback = src;
+    src      = config.paths.source.definitions + '/**/' + config.globs.components + '.less';
+  }
+
+  const func       = (callback) => buildCSS(src, 'docs', config, {}, callback);
+  func.displayName = "CSS Docs";
+
+  func(callback);
+}
+
+// Default tasks
+module.exports = rtlAndNormal;
+
+// We keep the changed files in an array to call build with all of them at the same time
+let timeout, files = [];
+
+/**
+ * Watch changes in CSS files and call the correct build pipe
+ * @param type
+ * @param config
+ */
+module.exports.watch = function (type, config) {
+  const method = type === 'docs' ? docs : rtlAndNormal;
+
+  // Watch theme.config file
+  gulp.watch([
+    normalize(config.paths.source.config),
+    normalize(config.paths.source.site + '/**/site.variables'),
+    normalize(config.paths.source.themes + '/**/site.variables')
+  ])
+    .on('all', function () {
+      // Clear timeout and reset files
+      timeout && clearTimeout(timeout);
+      files = [];
+      return gulp.series(method)();
+    });
+
+  // Watch any less / overrides / variables files
+  gulp.watch([
+    normalize(config.paths.source.definitions + '/**/*.less'),
+    normalize(config.paths.source.site + '/**/*.{overrides,variables}'),
+    normalize(config.paths.source.themes + '/**/*.{overrides,variables}')
+  ])
+    .on('all', function (event, path) {
+      // We don't handle deleted files yet
+      if (event === 'unlink' || event === 'unlinkDir') {
+        return;
+      }
+
+      // Clear timeout
+      timeout && clearTimeout(timeout);
+
+      // Determine which LESS file has to be recompiled
+      let lessPath;
+      if(path.indexOf('site.variables') !== -1)  {
+        return;
+      } else if (path.indexOf(config.paths.source.themes) !== -1) {
+        console.log('Change detected in packaged theme');
+        lessPath = replaceExt(path, '.less');
+        lessPath = lessPath.replace(tasks.regExp.theme, config.paths.source.definitions);
+      } else if (path.indexOf(config.paths.source.site) !== -1) {
+        console.log('Change detected in site theme');
+        lessPath = replaceExt(path, '.less');
+        lessPath = lessPath.replace(config.paths.source.site, config.paths.source.definitions);
+      } else {
+        console.log('Change detected in definition');
+        lessPath = path;
+      }
+
+      // Add file to internal changed files array
+      if (!files.includes(lessPath)) {
+        files.push(lessPath);
+      }
+
+      // Update timeout
+      timeout = setTimeout(() => {
+        // Copy files to build in another array
+        const buildFiles = [...files];
+        // Call method
+        gulp.series((callback) => method(buildFiles, callback))();
+        // Reset internal changed files array
+        files = [];
+      }, 1000);
+    });
+};
+
+// Expose build css method
+module.exports.buildCSS = buildCSS;
```

Comment:
```suggestion
module.exports.buildCSS = buildCSS;

```

Reply:
Unless it's deliberately not there of course ...

Reply:
Not there in the original, but I guess we can have it just to satisfy editorconfig:

```
$ cat node_modules/fomantic-ui/tasks/build/css.js | hexdump -C | tail -2
000021f0  75 69 6c 64 43 53 53 3b                           |uildCSS;|
000021f8
```